samples/BenchmarkDotNet.Samples/JIT/Jit_Inlining.cs

```diff
@@ -5,6 +5,9 @@ namespace BenchmarkDotNet.Samples.JIT
 {
     // See http://en.wikipedia.org/wiki/Inline_expansion
     // See http://aakinshin.net/en/blog/dotnet/inlining-and-starg/
+#if !CORE
+    [Diagnostics.Windows.Configs.InliningDiagnoserConfig]
```

Comment:
Both options looks bad. =( Ok, let keep the current behavior without obsolete attributes.
By the way, is it possible to implement diagnosers for .NET Core?